src/Core/src/Platform/Windows/NavigationRootManager.cs

```diff
@@ -120,7 +146,6 @@ public virtual void Disconnect()
 			}
       
 			SetToolbar(null);
-			SetTitleBar(null, null);
```

Comment:
Can we add this call up into the WindowHandler.Disconnect code? 

I think it's still useful to set the TitleBar to null if the handler itself is being disconnected

so we can add it here
![image](https://github.com/user-attachments/assets/61947571-3173-4c77-a650-2831efb68029)

after the disconnect call